docs/multi-cluster-ingress.md

```diff
@@ -1,18 +1,18 @@
 # Multi-cluster Ingress
 
-Users can use [MultiClusterIngress API](https://github.com/karmada-io/karmada/blob/master/pkg/apis/networking/v1alpha1/ingress_types.go) provided in Karmada to import external traffic to services in the member clusters.
+Users can use [Multi-cluster Ingress APIs](https://github.com/karmada-io/karmada/blob/master/pkg/apis/networking/v1alpha1/ingress_types.go) provided in Karmada to import external traffic to services in the member clusters.
```

Comment:
`MultiClusterIngress` is our crd name:

https://github.com/karmada-io/karmada/blob/6bd9f68a0d98452045938a06e2c6450d2fbf8018/pkg/apis/networking/v1alpha1/ingress_types.go#L27

I think there is no need to update it.